src/config/start.ts

```diff
@@ -54,6 +55,7 @@ export type StartCliFlags = Arguments<
     legacyMode: boolean;
     assetsFolder?: string;
     functionsFolder?: string;
+    forkProcess: boolean;
```

Comment:
could we prefix this with `experimental`? Like `--experimental-fork-process` it follows the same schema that Node.js uses for experimental features and it sets the expectation. Because that way people shouldn't be surprised if this flag later disappears

Reply:
Good idea, didn't think about making it obviously experimental.